modules/mobile/app/models/mobile/v0/adapters/claim_letter_documents.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Adapters
+      class ClaimLetterDocuments
+        def self.parse(documents)
+          documents.map do |document|
+            Mobile::V0::ClaimLetterDocuments.new(
+              # Backwards compatibility: the document ids that came from eFolder were wrapped in {} braces
```

Comment:
Can we add logic to check and see if the brackets exist already or are we confident they're being added only in the VBMS flow?

Reply:
According to the documentation for Lighthouses Benefits Document service they do not exist. Though we could add logic in here to check for them first.

Reply:
@jmarchi I've added a check just in case. Better safe than sorry. Thanks for looking that over!